sonarqube/resource_sonarqube_project.go

```diff
@@ -19,19 +19,19 @@ type Project struct {
 
 // GetProject for unmarshalling response body from getting project details
 type GetProject struct {
-	Paging     Paging              `json:"paging"`
```

Comment:
Are you sure about this ? There is a lot of changes done here and removing paging why ?

You are also removing fields that used to exist, are they not there anymore because of changes to the api or ?

Reply:
Nice find. Is there an enumerated list of resources supported by `components/show`? Wondering where else we could use that.

Reply:
It's pretty vague...

![bild](https://user-images.githubusercontent.com/34520175/228231153-925c15f2-e4c6-4ccf-b4fb-60b11bb76be0.png)